tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Here please follow this setup, https://github.com/verdaccio/verdaccio-audit/blob/master/tsconfig.json

```
{
	"compilerOptions": {
		"target": "esnext",
		"module": "commonjs",
		"declaration": true,
		"noImplicitAny": false,
		"strict": true,
		"outDir": "lib",
		"allowSyntheticDefaultImports": true,
		"esModuleInterop": true,
		"typeRoots": [
			"./node_modules/@verdaccio/types/lib/verdaccio",
			"./node_modules/@types"
		]
	},
	"include": [
		"src/*.ts",
		"types/*.d.ts"
	]
}
```

Some types come from `@verdaccio/types` library. This does not mean we cannot extend it, but, we know the setup above just works. Target must be `"target": "esnext",`.